e2e/curl_nginx.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       serviceAccountName: test
       containers:
         - name: nginx
-          image: nginx:1.10
+          image: nginx:1.21.6
```

Comment:
maybe we should use a specific SHA to ensure that they don't push a replacement for this tag again?